irctest/cases.py

```diff
@@ -155,7 +155,10 @@ class BaseServerTestCase(_IrcTestCase):
     def setUp(self):
         super().setUp()
         self.find_hostname_and_port()
-        self.controller.run(self.hostname, self.port)
+        kwargs = {}
+        if self.server_start_delay is not None:
+            kwargs['start_wait'] = self.server_start_delay
+        self.controller.run(self.hostname, self.port, **kwargs)
```

Comment:
`self.server_start_delay` defaults to None, since the various servers set their own default start delays (for instance, Insp and mammon are 0.1 and 0.5 respectively). This lets them use the default values in their def init, eg:
```
def run(self, hostname, port, start_wait=0.5):
```